### ISSUE TITLE
white screen reveals when windowManager.show after windowManager.hide

### ISSUE THREAD

**Issue by Iuppit2r** [leanflutter/window_manager#155]
white screen reveals (up to 15 secs) when windowManager.show after windowManager.hide. (windows)

**Comment by lijy91** [comment]
As a temporary solution, call the setState method once after the window is reshown.

**Comment by sufu777** [comment]
> As a temporary solution, call the setState method once after the window is reshown.

I tried like this, but it seems not work correctly every time

**Comment by jakub-stefaniak** [comment]
Yeah, I have the same problem as @sufu777. Sometimes the setState workaround works, sometimes not.

**Comment by damywise** [comment]
Related: #131

**Comment by lijy91** [comment]
window_manager 0.5.x is no longer maintained: from 0.6.0 window_manager is rebuilt on [nativeapi](https://github.com/libnativeapi/nativeapi) (Flutter package: [nativeapi](https://pub.dev/packages/nativeapi)), and its platform code moves there.

This hasn't been checked against nativeapi yet, so it's now on the verification list in libnativeapi/nativeapi#75. If you can reproduce it with nativeapi's `Window` API (or window_manager 0.6.0 once released), please add your reproduction there.
